_pages/introduction.md

```diff
@@ -0,0 +1,124 @@
+
+Regulations.gov provides [a summary of the steps of the regulatory (rulemaking) process](http://www.regulations.gov/#!home;tab=learn), the Department of Transportation provides [a more in-depth overview of the process](https://www.transportation.gov/regulations/rulemaking-process), and Reginfo.gov provides [a detailed chart of the process (the "Reg Map")](http://www.reginfo.gov/public/reginfo/Regmap/index.jsp).
+
+There are a few different types of Federal Register Notices that can be part of a notice and comment process:
+
+### Advanced Notice of Proposed Rulemaking (ANPR)
+
+This doesn’t happen as often as the following notices. This usually happens before any change is made to a regulation. An ANPR is put out when an agency wants to check if an idea for future regulation change is something the public may want. It is meant to get early feedback from the public, and the issues in this notice may not move forward to the next notice depending on the public’s response.
+
+### Notice of Proposed Rulemaking (NPR) or Proposed Rule
+
+This is the initial draft of a change to a regulation. The notice starts with a “Preamble” which can contain the reasons for the regulation change, some section-by-section analysis, a cost analysis, and other general explanations. The Preamble is followed by the changes to the regulation. This notice only shows the actual changed parts of the regulation, not the new full text of the regulation.
+
+### Final Rule
+
+This is the final draft of a change to a regulation. This notice also starts with a Preamble and is followed by the changed parts of the regulation (not in context of the full text).
+
+### Interim Final Rule
+
+This is the same as a final rule except that it does not have to follow a proposed rule. This is uncommon. An example of this kind of rule: when CFPB was created, it reissued all of the regulations it was given authority over in full as interim final rules. There was no need for a notice and comment period.
+
+## <a name="structure"></a>The structure of a regulation
+
+Each regulation is part of the careful hierarchical structure of the [Code of Federal Regulations](https://www.gpo.gov/fdsys/browse/collectionCfr.action?collectionCode=CFR), and eRegulations reflects this structure. Here's a guide from the top down.
+
+### The Code of Federal Regulations contains “titles”
+
+The Code of Federal Regulations is broken into numeric “titles”, which each describe one subject area (such as “Domestic Security” or “Banks and Banking”). Agencies will most likely only cover one title. (1-50 is usually written in Roman numerals.)
```

Comment:
> (1-50 is usually written in Roman numerals.)

Maybe "The first fifty titles are often identified with Roman numerals, e.g. 'Title XXVII'"